Assets/Scripts/PlayerController.cs

```diff
@@ -15,26 +16,29 @@ public class PlayerController : MonoBehaviour {
     private CharacterController _controller;
     private Vector3 _velocity;
     private Animator _animator;
+    private PlayerInputManager _inputManager;
 
     void Start ()
     {
         _controller = GetComponent<CharacterController>();
         _animator = GetComponent<Animator>();
+        _inputManager = GetComponent<PlayerInputManager>();
     }
 
     void Update ()
     {
-        if (Input.GetButton("Jump") && _controller.isGrounded)
+        if (_inputManager.IsButtonDown(PlayerInputManager.Key.Jump) && _controller.isGrounded)
         {
             _velocity.y += Mathf.Sqrt(JumpHeight * -2f * Gravity);
             Debug.Log("jump");
             _animator.SetTrigger("Jump");
         }   
+        var horizontalInput = _inputManager.GetAxis(PlayerInputManager.Key.MoveHorizontal);
+        Vector3 move = new Vector3(horizontalInput, 0, 0);
+        _animator.SetFloat ("Speed", Mathf.Abs(horizontalInput));
+        _animator.SetBool ("IsGrounded", _controller.isGrounded);
 
-        Vector3 move = new Vector3(Input.GetAxis("Horizontal"), 0, 0);
-        _animator.SetFloat ("Speed", Mathf.Abs(Input.GetAxis("Horizontal")));
-
-        if (Input.GetButtonDown("Dash"))
+        if (_inputManager.IsButtonDown(PlayerInputManager.Key.Dash))
         {
             Debug.Log("Dash");
```

Comment:
On line 61 it's commented out - why is this one active?

Reply:
It's useful when debugging... Commented out to remove console spam. Yeah probably should remove most of Debug logging later.